hostthedocs/filekeeper.py

```diff
@@ -15,15 +15,46 @@ def sort_by_version(x):
     return x['version'].replace('.', '~') + 'z'
 
 
+def is_valid_doc_folder(folder):
```

Comment:
Rename to `_is_valid_doc_root_folder`, it may as well have a protected name, and it's more clear this only applies to 'roots' and not nested doc folders (this tripped me up reading over the code).